.drone.star

```diff
@@ -2367,6 +2381,7 @@ def runWebuiAcceptanceTests(ctx, suite, alternateSuiteName, filterTags, extraEnv
         "image": OC_CI_NODEJS,
         "environment": environment,
         "commands": [
+            "ls /usr/src/app/filesForUpload",
```

Comment:
I guess this is helpful for easily debugging if something goes wrong with filesForUpload